src/components/program-progress/ProgramListingCard.jsx

```diff
@@ -0,0 +1,138 @@
+import { breakpoints, Card } from '@edx/paragon';
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+import { AppContext } from '@edx/frontend-platform/react';
+
+const ProgramListingCard = ({ program }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const history = useHistory();
+
+  const handleCardClick = () => {
+    history.push(`/${enterpriseConfig.slug}/program-progress/${program.uuid}`);
+  };
+
+  useEffect(() => {
+    const handleWindowResize = () => {
+      setWindowWidth(window.innerWidth);
+    };
+
+    window.addEventListener('resize', handleWindowResize);
+
+    return () => {
+      window.removeEventListener('resize', handleWindowResize);
+    };
+  }, []);
+
+  const getBannerImageURL = () => {
+    let imageURL = '';
+    if (windowWidth >= breakpoints.large.minWidth) {
+      imageURL = program.bannerImage.large.url;
+    } else if (windowWidth >= breakpoints.medium.minWidth) {
+      imageURL = program.bannerImage.medium.url;
+    } else if (windowWidth >= breakpoints.small.minWidth) {
+      imageURL = program.bannerImage.small.url;
+    } else {
+      imageURL = program.bannerImage.xSmall.url;
+    }
+    return imageURL;
+  };
+
+  return (
+    <Card className="mb-4 program-listing-card mr-5" onClick={handleCardClick}>
+      <Card.Img
+        src={getBannerImageURL()}
+        data-testid="program-banner-image"
+      />
+      {(program.authoringOrganizations?.length === 1 && program.authoringOrganizations[0].logoImage) && (
+        <div className="partner-logo-wrapper shadow-sm">
+          <img
+            src={program.authoringOrganizations[0].logoImage}
+            className="partner-logo"
+            alt={program.authoringOrganizations[0].key}
+          />
+        </div>
+      )}
+      <Card.Body>
+        <div className="font-weight-light d-flex justify-content-between">
+          <div>
+            {program.authoringOrganizations?.length > 0 && program.authoringOrganizations.map(org => org.key).join(' ')}
+          </div>
+          <div>
+            {program.type}
+          </div>
+        </div>
+        <h3 style={{ color: '#454545' }}>
```

Comment:
I think this should be added in css fiee?

Reply:
done